src/components/Button.tsx

```diff
@@ -4,41 +4,42 @@ import styled from 'styled-components';
   FOR PRIMARY BUTTON USAGE:
 
     Props: 
-      primaryColor (Required) - This color will determine the default background color of the button
-      secondaryColor (Required) - This color will determine the border color and background color on hover
+      $primaryColor (Required) - This color will determine the default background color of the button
+      $secondaryColor (Required) - This color will determine the border color and background color on hover
    
     Example:
-      <Button primaryColor={COLORS.primaryBlue} secondaryColor={COLORS.primaryBlueClicked} onClick={...}>
+      <Button $primaryColor={COLORS.blueMid} $secondaryColor={COLORS.blueDark} onClick={...}>
         [Button text here]
       </Button>
 
   FOR SECONDARY BUTTON USAGE:
 
     Props:
-      primaryColor (Omit) - IMPORTANT: DO NOT PASS IN A primaryColor PROP!!! The background color will be white by default
-      secondaryColor (Required) - This color will determine the border color and background color on hover
+      $primaryColor (Omit) - IMPORTANT: DO NOT PASS IN A primaryColor PROP!!! The background color will be white by default
+      $secondaryColor (Required) - This color will determine the border color and background color on hover
 
     Example: 
-      <Button secondaryColor={COLORS.primaryBlue} onClick={...}>
+      <Button $secondaryColor={COLORS.blueMid} onClick={...}>
         [Button text here]
       </Button>
  */
 const Button = styled.button<{
-  primaryColor?: string;
-  secondaryColor: string;
+  $primaryColor?: string;
+  $secondaryColor: string;
 }>`
   appearance: none;
-  color: ${props => (props.primaryColor ? 'white' : 'black')};
-  background: ${props => (props.primaryColor ? props.primaryColor : 'white')};
+  color: ${props => (props.$primaryColor ? 'white' : 'black')};
+  background: ${props => (props.$primaryColor ? props.$primaryColor : 'white')};
   align-self: flex-end;
```

Comment:
since the regular button will be used throughout the app, should we remove the `align-self` property? it may be best to let the container of the button control its positioning